src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -565,6 +566,165 @@ JSON example:
+| BackgroundSuppression             | REQUIRED                                                                                                                        | [boolean][]                             | Boolean indicating if background suppression is used.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| M0                                | REQUIRED                                                                                                                        | [number][] or [string][] or [boolean][] | Describes the available M0 information or the location of an `m0scan` file, as either: Numerical: a single numerical whole-brain M0 value, for example obtained from CSF in another sequence; SeparateFile: `*_m0scan.nii[.gz]`; Within ASL time-series: Boolean: `true` referring to an `m0scan` within the time-series (as defined within the `*_aslcontext.tsv`); `false` referring to the use of the (average) `control` image for M0 calibration.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Is the `asl` file's `M0` not redundant with the `m0scan`'s `IntendedFor`?

Reply:
Hi @tsalo, we are rethinking the M0 field, based on an earlier comment. This redundancy will probably be solved with the new approach. I'll add it once we finished it and link to you.

Reply:
@tsalo We updated it using the suggestions. Feel free to take a look.